pina/loss/ntk_weighting.py

```diff
@@ -0,0 +1,61 @@
+"""Module for Neural Tangent Kernel Class"""
+
+import torch
+from torch.nn import Module
+from .weighting_interface import WeightingInterface
+from ..utils import check_consistency
+
+
+class _NoWeighting(WeightingInterface):
+    def aggregate(self, losses):
+        return sum(losses.values())
+
+
+class NeuralTangetKernelWeighting(WeightingInterface):
+    """
```

Comment:
doc missing, see other weightings to follow the same standard. Here a great example:
https://github.com/mathLab/PINA/blob/15ac30b19dcd5f1db69dc6f6668b08f669ef2ab2/pina/loss/scalar_weighting.py#L23

Reply:
Also the original reference must be included in the doc. See https://github.com/mathLab/PINA/blob/15ac30b19dcd5f1db69dc6f6668b08f669ef2ab2/pina/problem/zoo/advection.py#L62